src/main/java/de/craften/plugins/rpgplus/components/ability/AbilityComponent.java

```diff
@@ -76,10 +77,11 @@ public void giveAbility(Player player, String identifier, long durationSeconds)
 
 		final long now = new Date().getTime();
 		AtomicBoolean renewed = new AtomicBoolean(false);
-		getStore(player).update(ability.getIdentifier(), (oldEndTimeString) -> {
+		getStore(player).update("abilitymanager." + ability.getIdentifier() + ".endTime", (oldEndTimeString) -> {

```

Comment:
This string concatenation should be moved into a static method, e.g. `private static String getStoreKey(Ability ability)` to make the code cleaner and reduce the typo potential.